azure-pipelines.yml

```diff
@@ -178,6 +192,14 @@ jobs:
 
   - script: |
       export MACOSX_DEPLOYMENT_TARGET=10.9
+      if [ "$(python.installer.universal2)" = true ] ; then
+              # We need to "fix" the platform tag if the Python installer is universal2
+              # See:
+              #     * https://github.com/pypa/setuptools/issues/2520
+              #     * https://github.com/neuronsimulator/nrn/pull/1562
+
+              export _PYTHON_HOST_PLATFORM="macosx-${MACOSX_DEPLOYMENT_TARGET}-x86_64"
+      fi
```

Comment:
`nitpick`: to be bit more robust - I will move this inside `nrn/packaging/python/build_wheels.bash` where you can run get output of `sysconfig.get_platform()` using Python being used and replace `-universal2` with `-x86_64`.